CMSgov/design-system
# [WNMGDS-1301] Color primary-alt to secondary

## Summary

- Updates color-secondary to be used where color-primary-alt was previously being used
- Updates component theme variables to use color-secondary
- Added notes for deprecation of color-primary-alt
- Updated current examples
- Looks like this one gets loki snapshot updates for the datepicker updates in master also
- Adds a color-secondary-lighter as color-secondary-darker exists, I think lighter should also, but let me know, it's a value between light and lightest.

**Note:** Updates to the new doc site will need to happen once we get the token feature branch merged, I think it's best to just do those all at once. I can write a new ticket for new doc site token cleanup for future work but this get's the base ready.

## How to test

`yarn install && yarn build && yarn build:healthcare && yarn build:medicare`
`yarn loki test && yarn loki:healthcare test && yarn loki:medicare test`
`yarn start each` check text colors, background colors, color swatches, buttons, drawers
